### PR TITLE
ES6 Module Functionalities for Awesome Book Project

### PR DESCRIPTION
**Pull request by Cosmosha**
The following  features and requirements were implemented;

- Convert functions into es6 arrow functions. 
- ES6 Module / Modules folder
- Import luxon;

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @Cosmosha,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] You didn't document your work professionally:
   - You forgot to edit your README file. Please use the Microverse README template and edit it with your personal and project data.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,176 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
```
Comment (omar-muhamad) [inline README.md]: - [ ] Kindly put your name here instead of `Author1` to make your README file look more professional.

File: README.md [hunk 2]
```diff
@@ -1 +1,176 @@
...
+- **[Project follows the right structure for Github flows]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]('#')
+
```
Comment (omar-muhamad) [inline README.md]: - [ ] Please remove the demo link or replace it with `Coming soon` to make your README file look more professional.

File: modules/book.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      this.bookTitle.value = '';
+    }
+
+    // eslint-disable-next-line max-len
+    saveBookToLocalStorage = () => {
+      window.localStorage.setItem(this.storageName, JSON.stringify(this.bookCollection));
+    };
+
+    // eslint-disable-next-line max-len, no-return-assign
+    removeBook = (item) => {
```
Comment (omar-muhamad) [inline modules/book.js]: - [ ] Please remove eslint disablement lines an fix these issues yourself.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Cosmosha,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- Your app styling is great. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by binhussen — APPROVED** [review]
Hi @Cosmosha , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**